scripts/compose.py

```diff
@@ -1813,14 +1816,13 @@ def start_handler(self):
                 pgdata={"driver": "local"},
             ),
         )
-        yaml_opts = dict(
-            explicit_start=True,
-        )
         docker_compose_path = args["docker_compose_path"]
-        yaml.dump(compose, docker_compose_path, **yaml_opts)
+        json.dump(compose, docker_compose_path)
```

Comment:
do you think it would make sense to change the default value for `docker_compose_path` to `docker-compose.json`?

https://github.com/elastic/apm-integration-testing/blob/f51456e53b1045afe96633d6eecd3de178d27afb/scripts/compose.py#L1629